src/components/FinishAccount/FinishAccount.jsx

```diff
@@ -25,42 +35,166 @@ const FinishAccount = ({ inviteId, data }) => {
       setErrorMessage(err.message);
     }
   };
+  */
+
+  const passwordRulesTooltipText =
+    '\u2022 Must be at least 8 characters long' +
+    '<br />' +
+    '\u2022 Include at least one uppercase and lowercase letter' +
+    '<br />' +
+    '\u2022 Include at least 1 number' +
+    '<br />' +
+    '\u2022 Include one of these set of special characters [@$!%*?&]';
+
+  const [newPasswordShown, setNewPasswordShown] = useState(false);
+  const toggleNewPasswordVisibility = () => {
+    setNewPasswordShown(!newPasswordShown);
+  };
+
+  const [reenterPasswordShown, setReenterPasswordShown] = useState(false);
+  const toggleReenterPasswordVisibility = () => {
+    setReenterPasswordShown(!reenterPasswordShown);
+  };
+
+  const passwordRegExp = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,255}$/;
+  const passValidationSchema = Yup.object().shape({
+    currentPassword: Yup.string().required('Current password is required'),
+    newPassword: Yup.string()
+      .required('Please enter your new password')
+      .matches(passwordRegExp, 'Password is invalid'),
+    reenterNewPassword: Yup.string()
+      .required('Please re-enter your new password')
+      .oneOf([Yup.ref('newPassword'), null], 'Passwords do not match!'),
+  });
+
+  const {
+    register: registerPass,
+    handleSubmit,
+    formState: { errors: errorsPass },
+  } = useForm({
+    resolver: yupResolver(passValidationSchema),
+  });
+
   return (
-    <div>
-      <h2>Finish Account Creation</h2>
-      {errorMessage && <p>{errorMessage}</p>}
-      {!confirmationMessage && (
-        <form onSubmit={handleResetPassword}>
-          <input placeholder="First Name" disabled value={data.firstName} />
-          <br />
-          <input placeholder="Last Name" disabled value={data.lastName} />
-          <br />
-          <input placeholder="Email" disabled value={data.email} />
-          <br />
-          <input placeholder="Position" disabled value={data.position} />
-          <br />
-          <input
-            type="password"
-            onChange={({ target }) => setPassword(target.value)}
-            placeholder="New Password"
-          />
-          <br />
-          <input
-            type="password"
-            onChange={({ target }) => setCheckPassword(target.value)}
-            placeholder="Re-enter Password"
-          />
-          <br />
-          <button type="submit">Set Password</button>
+    <Container>
+      <div className={styles['logo-wrapper']}>
+        <img src={logo} alt={logo} className={styles.logo} />
+      </div>
+      <Col md={{ span: 8, offset: 2 }}>
+        <form className={`form-group ${styles['site-form']}`} onSubmit={handleResetPassword}>
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'handleResetPassword' is not defined.